src/sql/workbench/contrib/objectExplorer/test/browser/connectionTreeActions.test.ts

```diff
@@ -79,6 +81,34 @@ suite('SQL Connection Tree Action tests', () => {
 		return connectionManagementService;
 	}
 
+	function createNotificationService(promptChoice: number): TypeMoq.Mock<INotificationService> {
+		let notificationService = TypeMoq.Mock.ofType<INotificationService>(TestNotificationService, TypeMoq.MockBehavior.Loose);
+		notificationService.callBase = true;
+		notificationService.setup(x => x.prompt(TypeMoq.It.isAny(), TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns((
+			severity: Severity,
+			message: string,
+			choices: IPromptChoice[],
+			options?: IPromptOptions
+		) => {
+			choices[promptChoice].run();
+			return {
```

Comment:
Would be simpler to just use NoOpNotification from src\vs\platform\notification\common\notification.ts